master/embedded.html

```diff
@@ -15,31 +15,6 @@
 <body>
 
 <h1>Embedded Content</h1>
-<div class="annotation svg2-requirement">
```

Comment:
Have we been removing annotations about deferred features? I would have expected to just add in information about the decision to defer.

Reply:
We keep those annotations in the 2.1 draft. So they are not lost. I still agree that we should keep annotation for moved, changed or added features?